README.md

```diff
@@ -1,2 +1,70 @@
-# enhancements
-Enhancement Tracking and Backlog Repo for Istio Releases
+# Enhancement Tracking and Backlog
+Enhancement Tracking and Backlog Repo for Istio Releases. Owned by TOC.
+
+This repo contains issues and Istio enhancement proposal. These issues are umbrellas for new enhancements to be added to Istio. An enhancement usually takes multiple releases to complete. And an enhancement can be tracked as backlog items before work begins. An enhancement may be filed once there is consensus in at least one Istio working group.
+
+## Why do we need this?
+
+As a project it is vital to be able to track the chain of custody for a proposed enhancement from conception through implementation. The enhancement repository provides a standardized mechanism for describing important enhancements delivered through Istio releases.  This will not only enable the contributors, maintainers and release managers to explaining features to users coherently but also allow adopters to plan their adoption strategy based on a forward looking road map.
```

Comment:
```suggestion
As a project it is vital to be able to track the chain of custody for a proposed enhancement from conception through implementation. The enhancement repository provides a standardized mechanism for describing important enhancements delivered through Istio releases. This will not only enable the contributors, maintainers and release managers to explaining features to users coherently but also allow adopters to plan their adoption strategy based on a forward looking road map.
```